Swiftification/ObserverSet.swift

```diff
@@ -124,7 +124,7 @@ extension ObserverSet: CustomStringConvertible {
             entry in
             (entry.object === self
                 ? "\(entry.f)"
-                : "\(entry.object) \(entry.f)")
+                : "\(String(describing: entry.object)) \(entry.f)")
```

Comment:
Agh, you know what, this one doesn't really matter. Let's just go with what you originally had.